eventarc/gke_basic/main.tf

```diff
@@ -0,0 +1,39 @@
+/**
+* Copyright 2023 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+# [START terraform_eventarc_enableapi]
```

Comment:
Hi Betsy, to clarify, did you mean to enable 
resource "google_project_service" "container" {
  provider           = google-beta
  service            = "container.googleapis.com"
  disable_on_destroy = false
}
in eventarc/gke_eventarc/main.tf and in eventarc/gke_pubsub_trigger/main.tf?

Reply:
If they use the service, then yes.

Don't use `provider = google-beta` unless you need to.
Don't enable services that aren't being used in a sample.
Conversely, do enable a service if the service is being used in a sample.

Reply:
Also don't include `provider = google` unless you need to.

Reply:
Thanks, Betsy!
Removed <code>provider = google</code from eventarc/gke_basic/main.tf and eventarc/gke_pubsub_trigger/main.tf.

Retained <code>provider = google-beta</code>  in the eventarc/gke_eventarc/[main.tf](http://main.tf/) because it produced an invalid token JSON error when I tried to run the file.